docs/source/ttnn/usage.rst

```diff
@@ -277,3 +276,40 @@ The following environment variable can be set in order to completely disable the
     ttnn.print_l1_buffers()
 
     ttnn.close_device(device)
+
+
+
+13. Register pre- and/or post-operation hooks
+---------------------------------------------
```

Comment:
I noticed in general we are leaving off the "Why" for the usage.  Should we start including that?

Reply:
I think we already include the why in the examples where it makes sense to do so. Other examples are more open-ended on the other hand and I don't think we should restrict people's imagination with our reasons. The examples themselves already show why something could be used